### PR TITLE
fix (BuilderEllipsoidTile) : fix wrong height in OBB computation

### PR DESCRIPTION
**Pull request by gchoqueux**
Bug introduce in PR #303 commit c839ad5a

### CODE REVIEW

File: src/Core/Prefab/Globe/BuilderEllipsoidTile.js [hunk 1]
```diff
@@ -113,9 +113,9 @@ BuilderEllipsoidTile.prototype.OBB = function OBBFn(params) {
     qRotY.multiply(planeZ);
 
     for (var i = 0; i < cardinals.length; i++) {
-        const cardinal3D = cardinals[i].as('EPSG:4978').xyz().sub(params.center);
-        cardin3DPlane.push(tangentPlaneAtOrigin.projectPoint(cardinal3D));
-        // compute height max
+        const cardinals3D = cardinals[i].as('EPSG:4978').xyz();
```
Comment (peppsac) [inline src/Core/Prefab/Globe/BuilderEllipsoidTile.js]: Could you keep the `cardinal3D` name please?

Comment (gchoqueux) [reply src/Core/Prefab/Globe/BuilderEllipsoidTile.js]: done

**Comment by gchoqueux** [comment]
@peppsac updated

File: src/Core/Prefab/Globe/BuilderEllipsoidTile.js [hunk 2]
```diff
@@ -113,9 +113,9 @@ BuilderEllipsoidTile.prototype.OBB = function OBBFn(params) {
     qRotY.multiply(planeZ);
 
     for (var i = 0; i < cardinals.length; i++) {
-        const cardinal3D = cardinals[i].as('EPSG:4978').xyz().sub(params.center);
+        const cardinal3D = cardinals[i].as('EPSG:4978').xyz();
         cardin3DPlane.push(tangentPlaneAtOrigin.projectPoint(cardinal3D));
-        // compute height max
+        vec.subVectors(cardinal3D, params.center);
```
Comment (peppsac) [inline src/Core/Prefab/Globe/BuilderEllipsoidTile.js]: Erf, missed this one: could you also remove the `vec` variable:
```js
cardinal3D.sub(params.center);
```

Comment (gchoqueux) [reply src/Core/Prefab/Globe/BuilderEllipsoidTile.js]: done

**Review by peppsac — APPROVED** [review]

**Pull request merged**